x-pack/plugins/security_solution_serverless/public/get_started/step_links/add_integration_button.tsx

```diff
@@ -23,22 +23,13 @@ const IntegrationsLinkComponent = () => {
     [navigateTo, integrationsUrl]
   );
   return (
-    <FormattedMessage
-      id="xpack.securitySolutionServerless.getStarted.togglePanel.configure.step3.description2.linkText"
-      defaultMessage="Go to {integrations} to ingest your own data!"
-      values={{
-        integrations: (
-          // eslint-disable-next-line @elastic/eui/href-or-on-click
-          <EuiLink onClick={onClick} href={integrationsUrl}>
-            <FormattedMessage
-              id="xpack.securitySolutionServerless.getStarted.togglePanel.configure.step3.description2.link"
-              defaultMessage="Integrations"
-            />
-          </EuiLink>
-        ),
-      }}
-    />
+    <LinkButton onClick={onClick} fill id="integrations" path="/browse/security">
```

Comment:
Try always to use the page IDs from the enums instead of hardcoding them. `onClick` and `path` are not needed:
```suggestion
    <LinkButton id={ExternalPageName.integrationsSecurity} fill>
```